source/includes/job-board/_applications.md

```diff
@@ -207,6 +216,7 @@ email | Applicant's email address
 *educations | An array of education objects. Each education object should have five fields: `school_name_id`, `degree_id`, `discipline_id`, `start_date`, and `end_date`. You can get the `school_name_id`, `degree_id`, `discipline_id` from our [List Schools](#list-schools), [List Degrees](#list-degrees), and [List Disciplines](#list-disciplines) endpoints. `start_date` and `end_date` will use a hash of month and year.
 *employments | An array of employments objects. Each employment object should have: `company_name`, `title`, `start_date`, and `current` (must be `true` or `false`). If `current` is `false`, must have `end_date`. `start_date` and `end_date` will use a hash of month and year.
 *demographic_answers | An array of demographic answer objects, applicable only if your organization has Greenhouse Inclusion and demographic questions are enabled on the job post. Each object must have a `question_id` field. The `answer_options` field is an array of objects, one for each `answer_option_id` the candidate selected. For answer options which support free-form responses, a `text` field may also be supplied with the candidate's hand-typed answer. Note that these questions are always optional, so the `answer_options` array may be empty, null, or omitted if the candidate did not make any selections.
+*data_compliance | An object representing an applicant's answer to required data compliance questions. This field is dependent on your organization's privacy and compliance configuration. Example: `{ "gdpr_consent_given": true }`.
```

Comment:
Small thing but generally we refer to people as "candidates" and not "applicants" but not sure if it matters :shruggy:

Reply:
ah I'll update it